docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-requirements.rst

```diff
@@ -10,22 +10,30 @@
 
 Deploying the Determined HPC Launcher on Slurm/PBS has the following requirements.
 
--  The login node, admin node, and compute nodes must be configured with a Linux distro compatible
-   with any of the following minimum distro versions: Red Hat® Enterprise Linux® 8, SUSE Linux
-   Enterprise Server 15, CentOS 7, Rocky Linux 8, Ubuntu 20.04. Additional minimum distro version
-   dependencies may be required by your choice of Slurm/PBS version and container runtime.
+-  The login node, admin node, and compute nodes must be installed and configured with one of the
+   following Linux distributions:
 
-   -  User and group configuration must be consistent across all nodes of the HPC cluster
```

Comment:
Is this no longer the case? !!

Reply:
Moved down to separate bullet so I could use sub-bullets for the distro list.